sdk/containerregistry/Azure.Containers.ContainerRegistry/README.md

```diff
@@ -78,28 +78,39 @@ We guarantee that all client instance methods are thread-safe and independent of
 
 ## Examples
 
-The following snippets show brief samples of common developer scenarios using the ACR SDK library. 
+The following snippets show brief samples of common developer scenarios using the ACR SDK library.
 Please note that each sample assumes there is a `REGISTRY_ENDPOINT` environment variable set to a string containing the `https://` prefix and the name of the login server, for example "https://myregistry.azurecr.io".
 
 ### Sync examples
 
-- [List repositories](#list-repositories)
-- [List tags with anonymous access](#list-tags-with-anonymous-access)
-- [Set artifact properties](#set-artifact-properties)
-- [Delete images](#delete-images)
+- Registry operations:
```

Comment:
Did we intentionally not add sync samples? I see sync and async for registry operations but only async ones for blob and manifest operations.

Reply:
Makes sense.